hazelcast-client-new/src/test/java/com/hazelcast/client/ClientConnectionTest.java

```diff
@@ -42,36 +41,37 @@
 
 import static org.junit.Assert.assertEquals;
 
-@RunWith(HazelcastSerialClassRunner.class)
+@RunWith(HazelcastParallelClassRunner.class)
 @Category(QuickTest.class)
 public class ClientConnectionTest extends HazelcastTestSupport {
 
+    private final TestHazelcastFactory hazelcastFactory = new TestHazelcastFactory();
```

Comment:
I am not really fond of that approach because I don`t find it fit to client. 
We have two kind of methods in HazelcastTestSupport

1) HazelcastInstance createHazelcastInstance(Config config);

This one create an instance and store it in HazelcastTestSupport. And we don`t use it ever again ?!?
For client I need to create at least two HazelcastInstance(one is server, other is client), 

2) TestHazelcastInstanceFactory createHazelcastInstanceFactory()
And for me this is not different than current one, this is just replacement for saying new TestHazelcastFactory()

I don`t really see any added value, and find current approach much more explicit. I might be missing something, if that is the case, I am listening.